test/static/typed_handle_common.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TYPED_HANDLE_ARGUMENT_COMMON_H
```

Comment:
test/static was deprecated a while ago, I thought. Did these files sneak in from old code?